docs/docs/reference/other-new-features/creator-applications.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: doc-page
+title: "Creator Applications"
+---
+
+Creator applications allow to use simple function call syntax to create instances
+of a class, even if there is no apply method implemented. Example:
+```scala
+class StringBuilder(s: String) {
+   def this() = this(s)
```

Comment:
Is `this(s)` a typo here? The compiler reasonably says that `s` is not accessible from the constructor.